enterprise/models.py

```diff
@@ -1519,7 +1519,7 @@ def __repr__(self):
         return self.__str__()
 
 
-def logo_path(instance, filename):
+def logo_path(instance, filename):  # pylint: disable=unused-argument
```

Comment:
i had to track down how/where the `logo_path` method gets called. i was confused that it was a free-floating method between classes (not inside a class in this file). could we move it inside the branding model class? or move it to a utility class?